test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -157,6 +164,48 @@ async function setup(t) {
   return challengeEmailPageObject;
 }
 
+test
+  .requestHooks(sendEmailMock)('send email screen should have right labels', async t => {
```

Comment:
can you also add a test that checks when this screen should not be shown.. probably add such check to any existing test..